### ISSUE TITLE
more space

### ISSUE THREAD

**Issue by AnonymousWebHacker** [vector-im/element-android#2272]
![IMG_20201020_001729](https://user-images.githubusercontent.com/43164089/96540455-cfadb300-126b-11eb-9201-3dcb77e86fd0.png)
![IMG_20201020_001652](https://user-images.githubusercontent.com/43164089/96540458-d2a8a380-126b-11eb-8d98-030ea302ee48.png)

**Comment by bmarty** [comment]
Yes thanks for having created the issue, we will improve this in the next Sprint.

**Comment by AnonymousWebHacker** [comment]
@bmarty There is only one thing left to close this 

*implemented (share / contacts / QR) 
![Screenshot_2021-01-20-19-59-41](https://user-images.githubusercontent.com/43164089/105259826-86b31680-5b5a-11eb-94a6-548c31b92bbf.png)

*But what if the list is 2000 users and I am well below?

![Screenshot_2021-01-20-20-00-34](https://user-images.githubusercontent.com/43164089/105259998-f1fce880-5b5a-11eb-9a79-5e214bfa219c.png)

It takes a lot of work to go through the list again, upwards, so that the QR options appear again ..etc

===================================================
* Posible Solution

If I am in the middle of the list, below, or where it is, (if I search up) it will come out (QR \ Share / Contacts) as a subjection
or implement some icons, that do not take up much space

And once again, thanks for element android
